### PR TITLE
Sandbox the simulator iframe.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**

### CODE REVIEW

File: src/simulator/Simulator.tsx [hunk 1]
```diff
@@ -111,6 +111,7 @@ const Simulator = ({
                 frameBorder="no"
                 scrolling="no"
                 allow="autoplay;microphone"
+                sandbox="allow-scripts allow-same-origin"
```
Comment (microbit-matt-hillsdon) [inline src/simulator/Simulator.tsx:114]: `allow-same-origin` is needed to use the Fetch API to download the WebAssembly.

**Comment by microbit-matt-hillsdon** [comment]
Equivalent simulator demo change: https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/64

**Comment by microbit-matt-hillsdon** [comment]
Tested in:
- [x] Firefox
- [x] Safari
- [x] Chrome
- [ ] Edge <- @microbit-robert please try, play some sounds as that seems like the most exciting thing we do.

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**